lib/integrations.rb

```diff
@@ -1,11 +1,16 @@
 # frozen_string_literal: true
 module Integrations
   class Error < StandardError
-    attr_reader :type, :message
+    attr_reader :type, :message, :response, :response_body, :response_code
 
-    def initialize(type='unknown_error', message='Unknown Error')
+    def initialize(type='unknown_error', message='Unknown Error', response)
       @type = type
       @message = message
+      if response && response.respond_to?(:body)
+        @response = response
```

Comment:
good point. i thought it may be good to have but it is not currently used. i'll remove.